firebase-ai/src/main/kotlin/com/google/firebase/ai/FirebaseAI.kt

```diff
@@ -212,15 +216,31 @@ internal constructor(
      * Returns the [FirebaseAI] instance for the provided [FirebaseApp] and [backend].
      *
      * @param backend the backend reference to make generative AI requests to.
+     * @param useLimitedUseAppCheckTokens when sending tokens to the backend, this option enables
+     * the usage of App Check's limited-use tokens instead of the standard cached tokens.
+     *
+     * A new limited-use tokens will be generated for each request; providing a smaller attack
+     * surface for malicious parties to hijack tokens. When used alongside replay protection,
+     * limited-use tokens are also _consumed_ after each request, ensuring they can't be used again.
+     *
+     * _This flag is set to `false` by default._
+     *
+     * **Important:** Replay protection is not currently supported for the FirebaseAI backend. While
```

Comment:
Is there anywhere else we can link about this? While the comment is useful, it can become outdated and make people think that updating the SDK is necessary to support it.

Reply:
Yeah, Rachel and the DevRel folks are working on adding a new section to the [App Check](https://firebase.google.com/docs/ai-logic/app-check) page. I can link to it, but I'm pretty sure it won't be out until Unity releases next release. Should I just link to it anyway, or should I leave a `TODO` that says to replace it with the link when the page gets updated?